TestON/tests/OnosCHO/OnosCHO.py

```diff
@@ -191,19 +191,19 @@ def CASE2( self, main ):
             main.log.report( "Controller assignment successfull" )
         else:
             main.log.report( "Controller assignment failed" )
-        time.sleep( 15 )
+        time.sleep( 20 )
```

Comment:
ONOS had issues in the past required a sleep here. Now it is ok and Kelvin's newCHO branch may not have this.